template/frameworks/vuetify/layouts/default.vue

```diff
@@ -110,7 +110,8 @@ export default {
       miniVariant: false,
       right: true,
       rightDrawer: false,
-      title: 'Vuetify.js'
+      title: 'Vuetify.js',
+      copyrightYear: new Date().getFullYear()
```

Comment:
You don't need another extra var, the interpolation `{{ new Date().getFullYear() }}` will work too